webpack.config.js

```diff
@@ -46,7 +46,7 @@ var config = [{
     './modules/conflict_resolver/js/resolved_conflicts_columnFormatter.js': './modules/conflict_resolver/jsx/resolved_conflicts_columnFormatter.js',
     './modules/conflict_resolver/js/unresolved_columnFormatter.js': './modules/conflict_resolver/jsx/unresolved_columnFormatter.js',
     './modules/examiner/js/columnFormatter.js': './modules/examiner/jsx/columnFormatter.js',
-    './modules/help_editor/js/columnFormatter.js': './modules/help_editor/jsx/columnFormatter.js',
```

Comment:
you shouldn't remove the columnFormatter entry if you're keeping it in a separate file

Reply:
I think it was discussed that the new standard is that columnFormatter should be a function within the parent component.